### PR TITLE
Feature/performance optimisation

### PR DESCRIPTION
**Pull request by 4d4mm**

### CODE REVIEW

File: packages/experience-editor/src/containers/ExperienceEditorApp.js [hunk 1]
```diff
@@ -192,15 +169,59 @@ class ExperienceEditorApp extends ManifestEditorApp {
...
+  toggleLoadManifestDialog = () => {
+    this.setState({
+      loadManifestDialogOpen: !this.state.loadManifestDialogOpen,
+    });
+  };
+
+  renderLoadManifestDialog = () => (
+    <LoadManifestModal
+      key={'LoadManifestDialog'}
+      collectionURL={'https://iiif-collection.ch.digtest.co.uk/p3/'}
```
Comment (stephenwf) [inline packages/experience-editor/src/containers/ExperienceEditorApp.js]: Should that come from props?

File: packages/experience-editor/src/containers/ExperienceEditorApp.js [hunk 2]
```diff
@@ -30,6 +29,9 @@ class ExperienceEditorApp extends ManifestEditorApp {
       renderer: this.renderPreviewDialog,
       openState: 'previewInContextDialogOpen'
     }];
+    if (!this.state.editorMode) {
+      this.state.editorMode = 'default';
```
Comment (stephenwf) [inline packages/experience-editor/src/containers/ExperienceEditorApp.js]: Should this be a set state call?

Comment (adam-digirati) [reply packages/experience-editor/src/containers/ExperienceEditorApp.js]: set state in constructor ?

Comment (stephenwf) [reply packages/experience-editor/src/containers/ExperienceEditorApp.js]: Ahh didn't notice it was there 👍

**Pull request merged**